ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -870,6 +870,7 @@ presubmits:
         - --dry-run=false
         - --prowjob-kubeconfig=/var/kubeconfigs/sa.pj-rehearse.app.ci.config
         - --kubeconfig-dir=/var/kubeconfigs
+        - --rehearsal-limit 10
```

Comment:
Let's keep the `--option=value` form please

Reply:
ack. Updated.